ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -0,0 +1,261 @@
+          env:
+            - name: ETCD_ENDPOINTS
+              valueFrom:
+                configMapKeyRef:
+                  name: calico-config
+                  key: etcd_endpoints
+            - name: ETCD_CA_CERT_FILE
+              valueFrom:
+                configMapKeyRef:
+                  name: calico-config
+                  key: etcd_ca
+            # Location of the client key for etcd.
+            - name: ETCD_KEY_FILE
+              valueFrom:
+                configMapKeyRef:
+                  name: calico-config
+                  key: etcd_key # Location of the client certificate for etcd.
+            - name: ETCD_CERT_FILE
+              valueFrom:
+                configMapKeyRef:
+                  name: calico-config
+                  key: etcd_cert
+            - name: IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+          volumeMounts:
+            - mountPath: /calico-secrets
+              name: etcd-certs
+          securityContext:
+            privileged: true
+      nodeSelector:
+        com.docker.ucp.kubernetes.calico/route-reflector: "true"
+      volumes:
+      # Mount in the etcd TLS secrets.
+        - name: etcd-certs
+          secret:
+            secretName: calico-etcd-secrets
+  ```
+
+Then, deploy the DaemonSet using:
+
+```
+kubectl create -f calico-rr.yaml
+```
+
+## Configure calicoctl
+
+To reconfigure Calico to use Route Reflectors instead of a node-to-node mesh,
+you'll need to SSH into a UCP node and download the `calicoctl` command.
+
+Log in to a UCP node using SSH, and run:
+
+```
+sudo curl --location https://github.com/projectcalico/calicoctl/releases/download/v3.1.1/calicoctl \
+  --output /usr/bin/calicoctl
+sudo chmod +x /usr/bin/calicoctl
+```
+
+Now you need to configure `calicoctl` to communicate with the etcd key-value
+store managed by UCP. Create a file named `/etc/calico/calicoctl.cfg` with
+the following content:
+
+```
+apiVersion: projectcalico.org/v3
+kind: CalicoAPIConfig
+metadata:
+spec:
+  datastoreType: "etcdv3"
+  etcdEndpoints: "127.0.0.1:12379"
+  etcdKeyFile: "/var/lib/docker/volumes/ucp-node-certs/_data/key.pem"
+  etcdCertFile: "/var/lib/docker/volumes/ucp-node-certs/_data/cert.pem"
+  etcdCACertFile: "/var/lib/docker/volumes/ucp-node-certs/_data/ca.pem"
+```
+
+## Disable node-to-node BGP mesh
+
+Not that you've configured `calicoctl`, you can check the current Calico BGP
+configuration:
+
+```
+sudo calicoctl get bgpconfig
+```
+
+If you don't see any configuration listed, create one by running:
+
+```
+cat << EOF | sudo calicoctl create -f -
+apiVersion: projectcalico.org/v3
+kind: BGPConfiguration
+metadata:
+  name: default
+spec:
+  logSeverityScreen: Info
+  nodeToNodeMeshEnabled: false
+  asNumber: 63400
+EOF
+```
+
+This creates a new configuration with node-to-node mesh BGP disabled.
+If you have a configuration, and `meshenabled` is set to `true`, update your
```

Comment:
`nodeToNodeMeshEnabled` is set to `true`

Reply:
To make it easier to follow the instructions, I first ask them to run the command without yaml output and without redirecting to a file. This makes it easy to figure out if `calicoctl` is misconfigured and fails to run, and makes it easier to see if there's a config or not, since it seems you won't have any configs with a default UCP deployment.

So this is what users will see.

```
NAME      LOGSEVERITY   MESHENABLED   ASNUMBER
```